README.md

```diff
@@ -1,71 +1,67 @@
 # Web Test Transformer
 
 - [Web Test Transformer](#web-test-transformer)
-  - [Overview](#overview)
+  - [Introduction](#introduction)
   - [Setup](#setup)
   - [Usage](#usage)
     - [Dry Run](#dry-run)
     - [CLI Flags](#cli-flags)
       - [Required Flags](#required-flags)
+    - [Examples](#examples)
+  - [Functional Overview](#functional-overview)
   - [Verification](#verification)
     - [All PASS Requirement](#all-pass-requirement)
   - [Issues](#issues)
 
-## Overview
+## Introduction
 Web Test Transformer is designed to transform HTML tests in Chromium source from the legacy
 [js-test.js](https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/web_tests/resources/js-test.js)
 framework to the current
 [testharness.js](https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/web_tests/resources/testharness.js)
 framework.
 
 
-View the design document at [go/web-test-transformer](go/web-test-transformer) (internal Google access only).
+View the design document at [go/web-test-transformer](http://go/web-test-transformer) (internal Google access 
+only).
 
 ## Setup
 
 `$git clone git@github.com:Ecosystem-Infra/web-test-transformer.git`
 
+`$cd web-test-transformer`
+
 `$npm install` to install dependencies.
 
 ## Usage
 
 Requirements:
-- You must be in `chromium/src/third_party/blink/web_tests` (for verification)
-- You must have chromium built. (for verification)
-- You are not running the tool on Windows (no path separator support). 
-
-
-This tool does its best job to meaningfully log all of the major actions it takes, as well as any 
-problems or errors it encounters along the way. For a large directory there will be a lot of output,
-but it should be easily searchable. 
-
-For a file, the tool will attempt to transform that file
-and then attempt to verify that file. No summary output is provided.
+- You are not running the tool on Windows (no path separator support, see https://github.com/Ecosystem-Infra/web-test-transformer/issues/11). 
+- You must be in `chromium/src/third_party/blink/web_tests` within the Chromium source tree (for verification)
+- You must have content_shell built. (for verification)
+Steps:
```

Comment:
Nit: some more words, like "Here are the high-level steps for using the tool to update a batch of tests"